pom.xml

```diff
@@ -154,7 +154,7 @@
             <id>image</id>
             <properties>
                 <image.publish.skip>false</image.publish.skip>
-                <image.tags>latest</image.tags>
```

Comment:
${project.version} for release version

Reply:
Only release version need to add `${project.version}` tag?

Reply:
done